docs/releases/2.0.txt

```diff
@@ -205,6 +205,17 @@ Validators
 Backwards incompatible changes in 2.0
 =====================================
 
+Handling of strings vs bytestrings
+----------------------------------
+
+In older Django releases, having to support native Python 2 strings in addition
```

Comment:
To support native Python 2 strings, older Django versions had to accept both bytestrings and unicode strings. Now that Python 2 support is dropped, bytestrings should only be encountered around input/output boundaries (handling of binary fields or HTTP streams, for example). You might have to update your code to limit bytestring usage to a minimum, as Django no longer accepts bytestrings in certain code paths.